### PR TITLE
{math}[GCCcore/6.4.0] muParser v2.2.5

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/muParser/muParser-2.2.5-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ bytecode and precalculating constant parts of the expression.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+# https://github.com/beltoforion/muparser/archive/v2.2.5.tar.gz
+source_urls = ['https://github.com/beltoforion/muparser/archive/']
+sources = [SOURCELOWER_TAR_GZ]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/m/muParser/muParser-2.2.5-GCCcore-6.4.0.eb]: Please add SHA256 checksum

File: easybuild/easyconfigs/m/muParser/muParser-2.2.5-GCCcore-6.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+sanity_check_paths = {
+    'files': ['include/muParser.h', 'lib/libmuparser.so'],
```
Comment (vanzod) [inline easybuild/easyconfigs/m/muParser/muParser-2.2.5-GCCcore-6.4.0.eb]: Is there a reason to drop the use of `SHLIB_EXT`?

File: easybuild/easyconfigs/m/muParser/muParser-2.2.5-GCCcore-6.4.0.eb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    'dirs': [],
+}
+
+parallel = 1
```
Comment (vanzod) [inline easybuild/easyconfigs/m/muParser/muParser-2.2.5-GCCcore-6.4.0.eb]: Why serial build? The `foss-2015a` version of the same build seem to be fine with parallel build...

File: easybuild/easyconfigs/m/muParser/muParser-2.2.5-GCCcore-6.4.0.eb [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+description = """
+ muParser is an extensible high performance math expression parser library
+ written in C++. It works by transforming a mathematical expression into
+ bytecode and precalculating constant parts of the expression.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+# https://github.com/beltoforion/muparser/archive/v2.2.5.tar.gz
```
Comment (vanzod) [inline easybuild/easyconfigs/m/muParser/muParser-2.2.5-GCCcore-6.4.0.eb]: Can you also remove this comment?

File: easybuild/easyconfigs/m/muParser/muParser-2.2.5-GCCcore-6.4.0.eb [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+source_urls = ['https://github.com/beltoforion/muparser/archive/']
+sources = [SOURCELOWER_TAR_GZ]
```
Comment (boegel) [inline easybuild/easyconfigs/m/muParser/muParser-2.2.5-GCCcore-6.4.0.eb]: this is wrong, should be `v%(version)s.tar.gz`?

**Comment by boegel** [comment]
@JackPerdue One more style issue (yes, we're quite picky now):

```
muParser-2.2.5-GCCcore-6.4.0.eb:28:59: E225 missing whitespace around operator
```

You can check yourself using `eb --check-style --from-pr 4851`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2541.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/fd501164564c1258fb0e6bcd24543b14 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2035.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/08345530d25d5f7cecaeb78d1ea2cc45 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @JackPerdue!

**Pull request merged**